packages/flutter/lib/src/services/text_input.dart

```diff
@@ -846,6 +847,127 @@ mixin TextSelectionDelegate {
 
   /// Whether select all is enabled, must not be null.
   bool get selectAllEnabled => true;
+
+  /// Cut current [selection] to Clipboard.
+  ///
+  /// If [cause] is [SelectionChangedCause.toolBar], the position of
+  /// [bringIntoView] to selection will be called and hide toolBar.
```

Comment:
It seems like Flutter doesn't capitalize Bar, we just use "toolbar" all lowercase.  Can you make that change here and everywhere else that it's written toolBar in this PR?